chapters/intro/intro.md

```diff
@@ -0,0 +1,108 @@
+---
+layout: default
+project: 000
+title: Introduction
+chapter: true
+---
+
+# Introduction
+
+**[Arie van Deursen] and [Rogier Slag].**<br/>
+*Delft University of Technology, The Netherlands*
+
+[arie van deursen]: http://avandeursen.com
+[rogier slag]: https://github.com/rogierslag
+
+
+
+_Delft Students on Software Architecture_ is a collection of architectural descriptions of open source software systems written by students from Delft University of Technology during a master-level course taking place in the spring of 2015.
```

Comment:
Maybe also add a link to the online studyguide of the tudelft?  http://www.studiegids.tudelft.nl/a101_displayCourse.do?course_id=31246